socket/src/commonTest/kotlin/mqtt/socket/SocketTests.kt

```
@@ -250,12 +250,12 @@ class SocketTests {
         }
     }
 
-    @ExperimentalUnsignedTypes
+    @ExperimentalUnsignedTypes  
     private suspend fun launchServer(server: TCPServer) : TCPServer {
         server.startServer()
         assertNotEquals(server.getListenPort(), server.port, "Server listen port is diferent")
         GlobalScope.launch {
-            server.getClientConnection()
+            server.getClientConnection({exp-> println(exp)})
```

I feel like we should throw the exception so it falls the test